### PR TITLE
Feature/boa unfork

### PR DESCRIPTION
**Pull request by helmo**
[various issues](https://www.drupal.org/project/issues?text=boa+unfork&projects=provision%2C+hosting%2C+eldir%2C+Hostmaster+%28Aegir%29%2C+Aegir+Hosting+Git%2C+Aegir+Hosting+tasks+extra%2C+Aegir+Hosting+Logs%2C+Hosting+Site+Backup+Manager)

### CODE REVIEW

File: platform/migrate.provision.inc [hunk 1]
```diff
@@ -77,11 +77,82 @@ function drush_provision_drupal_provision_migrate($platform, $new_name = NULL) {
...
+          $new_aliases[] = str_replace($raw_old_uri, $raw_new_uri, $alias);
+        }
+      }
+      $unique_aliases = array_unique($new_aliases); // Make sure there are no duplicates.
+      $options['aliases'] = array(); // Reset original aliases array.
+      $options['aliases'] = $unique_aliases; // Use rewritten aliases array.
+      if (!is_null(d($platform)->name) && (d($platform)->name == d()->platform->name)) {
+        drush_set_option('rename_only', TRUE);
+        foreach ($new_aliases as $alias) {
+          drush_log(dt("DEBUG RENAME new_aliases is @this", array('@this' => $alias)));
```
Comment (jonpugh) [inline platform/migrate.provision.inc]: Let's remove these DEBUG logs.

Comment (jonpugh) [reply platform/migrate.provision.inc]: I removed them in   90ba4d6

**Comment by jonpugh** [comment]
Should add tests before we call this one done:
- [ ] Clone Task 
- [ ] use client email option, check user1 address.

**Comment by helmo** [comment]
@jonpugh can you create a new issue to add these extra tests?

**Comment by helmo** [comment]
for reference @jonpugh already started a PR for a clone test in https://github.com/aegir-project/tests/pull/11

**Pull request merged**